.travis.yml

```diff
@@ -19,7 +19,7 @@ sudo: required
 
 # Before anything else, get the latest versions of things
 before_script:
-  - julia -e 'using Pkg; Pkg.add([PackageSpec(name="BinaryBuilder", rev="master"), PackageSpec(name="BinaryProvider")])'
+  - julia -e 'using Pkg; Pkg.add([PackageSpec(name="BinaryBuilder", rev="master"), PackageSpec(name="MbedTLS", version=v"0.6.6"), PackageSpec(name="BinaryProvider")])'
```

Comment:
I assume the MbedTLS change is https://github.com/JuliaWeb/MbedTLS.jl/issues/193? A comment should be left here. Also, we should try not using the master version of BinaryBuilder.

Reply:
Done.